12

```diff
@@ -0,0 +1 @@
+Saved working directory and index state On master: grouped_mvp_actual only group those with width
```

Comment:
Not sure what this is about; just a weird git artifact?

Reply:
I have no idea how that happened, but removed it [bda751e](https://github.com/dimagi/commcare-hq/pull/33333/commits/bda751ed3320f2e98a14209e5e977803a9ec4c36)